pkg/osbuild/sfdisk_stage_test.go

```diff
@@ -41,3 +41,22 @@ func TestNewSfdiskStage(t *testing.T) {
 	actualStageDOS := NewSfdiskStage(&options, device)
 	assert.Equal(t, expectedStage, actualStageDOS)
 }
+
+func TestNewSfdiskStageInvalid(t *testing.T) {
```

Comment:
(super nitpick) I personally would combine this commit and the previous one (i.e. have code and test in the same commit)

Reply:
Squashed.